eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -47,9 +54,20 @@ def run_pipeline(self):
 
         for idx, func in enumerate(pipeline[pipeline_index:]):
             self.backend_data["pipeline_index"] = pipeline_index + idx
-            result = func(**self.backend_data) or {}
-            self.backend_data.update(result)
+            try:
+                result = func(**self.backend_data) or {}
+            except PearsonBaseError as pearson_error:
+                # clean kwargs to dont finish next pipeline launch.
+                executed_pipeline_kwargs = remove_keys_from_dict(self.backend_data, ["pipeline_index"])
+                tasks = importlib.import_module("eox_nelp.pearson_vue.tasks")
```

Comment:
just asking why not `from eox_nelp.pearson_vue.tasks import rti_error_handler_task
`

Reply:
At the top of the file is a circular python import error.
And I could add there `from eox_nelp.pearson_vue.tasks import rti_error_handler_task` but I want to avoid 
 `import-outside-toplevel`